src/Cache.js

```diff
@@ -7,57 +7,42 @@ const decode = (value) => {
   return value
 }
 
-// Keys are namespaced by db name, since in orbit each store gets its own cache
-const keyWrap = (prefix, key) => `${prefix}_${key}`
+// Maps keys from orbit-db v22 to keys currently in our implementation
+const keyMap = (prefix, key) => {
+  const splits = key.split('/')
+  const heads = key.includes('Heads')
+  return `${splits[2]}/${splits[3]}_${heads ? splits[4] : key}`
+}
```

Comment:
I see, so prefix is simply ignored here. Then we can remove it and `this.odbAddress` everywhere keyMap is called?

Reply:
Ah yup, thanks! removing